src/components/RadioButton.js/index.js

```diff
@@ -0,0 +1,32 @@
+import React, { Children } from 'react';
+import classNames from 'classnames';
+import styles from './style.module.css';
```

Comment:
You can also check how to use css modules [here](https://github.com/css-modules/css-modules/blob/master/README.md)